sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2824,14 +2824,17 @@ def teardown_no_lock(self,
                          terminate: bool,
                          purge: bool = False,
                          post_teardown_cleanup: bool = True,
-                         refresh_cluster_status: bool = True) -> bool:
+                         refresh_cluster_status: bool = True) -> None:
```

Comment:
Remove the return statement at L2854?

(This would have been caught by #1564.. I should get to reviewing that PR too 😅)

Reply:
Oops. Thank you for catching this! Fixed.